src/helpers/ImageOptimizer.js

```diff
@@ -68,6 +68,16 @@ const createImageOptimizer = (globalOpts) => {
   };
 };
 
+function resolveImageOnFileSystem(globalOpts, src) {
+  if (typeof globalOpts.imageBasePath == 'function') {
+    return globalOpts.imageBasePath(src);
+  }
+  if (typeof globalOpts.imageBasePath == 'string') {
+    return path.join(globalOpts.imageBasePath, src);
+  }
+  src = path.join(WORKING_DIR, src);
```

Comment:
Should be `return path.join(WORKING_DIR, src)`.

Reply:
Good catch! Fixed!